tests/customer_group/setup.py

```diff
@@ -160,3 +160,22 @@ def send_request_of_check_if_customer_group_can_be_deleted_with_group_id(group_i
 
     TestData.response_status_code = response.status_code
     return response.json()
+
+
+def send_request_of_update_code_of_customer_group(group_id, code_group, tax_class_id, token=None):
```

Comment:
rename `send_request_of_update_code_of_customer_group` to  `send_request_of_update_customer_group` because this is aupdating two diferent values